test/interactions/clickInteractionTests.ts

```diff
@@ -2,156 +2,186 @@
 
 describe("Interactions", () => {
   describe("Click Interaction", () => {
```

Comment:
The name followed by the type, so `Interactions.Click` --> "Click Interaction". The `Plot`s follow a similar pattern.